src/sentry/static/sentry/app/components/projectSelector.jsx

```diff
@@ -79,11 +85,18 @@ class ProjectSelector extends React.Component {
   }
 
   getProjects() {
-    const {organization, projects} = this.props;
-    const projectList =
-      projects || organization.projects.filter(project => project.isMember);
+    const {organization, projects, multiProjects} = this.props;
+
+    if (multiProjects) return multiProjects;
```

Comment:
This conditional should have braces. I'm going to enable curly: all mode in #12539